<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entrance</class>
 <widget class="QMainWindow" name="Entrance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Edge Screen - 欢迎</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: white;
}

QPushButton{
color: white;
border-color: #2e6da4;
background-color: #337ab7;
border-radius:20px;
}
QPushButton:hover{
background-color:#286090;
border-color:#204d74;
}
QPushButton:checked{
background-color:#286090;
border-color:#204d74;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>开始时政播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始视频播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="picsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始图片播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="versionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>780</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Version Info - Loading</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="aqi_input">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AQI_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AQI</string>
    </property>
   </widget>
   <widget class="QLabel" name="Refresh_Label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Refresh_input">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Regular</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="linkStartButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>400</width>
      <height>360</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Link
Start!!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>show_news()</slot>
 </slots>
</ui>
